tests/dbErrorHandlers.test.js

```diff
@@ -0,0 +1,49 @@
+const dbErrorHandlers = require('../src/dbErrorHandlers.js')
+const { messages: errMessages, names: errNames, codes: errCodes } = require('../src/constants/errors.json')
+const { buildNextSpyTest, buildBoomSpyTest } = require('./helpers/index.js')
+
+const tape = require('tape')
+
+tape('dbErrorHandlers.custom with non custom error', t => {
+  buildNextSpyTest(t, dbErrorHandlers.custom, 'test error message', 'not a custom error')
```

Comment:
Okay, this is quite hard to read, or work out what the hell is going on, yes, it is good to keep tests dry, but also they should be very easy to read.